### DISCUSSION TITLE
Does pyenv support "--enable-optimizations"?

### DISCUSSION THREAD

**Discussion by swamper123** [pyenv/pyenv#1998] [Q&A] [answered]
Hello together,

for an application I use, I want to install a python version with "--enable-optimizations".

On foot, when you compile python from source files, you are able to execute this in the installation stept:

`./configure --enable-optimizations`

Is there a way to tell pyenv to install such a version or do I have to muddle something on my own in that case?

**Comment by joshfriend** [comment, marked as answer]
This is documented in the readme [here](https://github.com/pyenv/pyenv/blob/master/plugins/python-build/README.md#special-environment-variables)

Use the `CONFIGURE_OPTS` variable, e.g. `CONFIGURE_OPTS="--enable-optimizations" pyenv install ...`
